mms/tests/unit_tests/test_validate_messages.py

```diff
@@ -0,0 +1,106 @@
+# Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+import pytest
+from mms.mxnet_model_service_error import MMSError
+from mms.utils.validators.validate_messages import ModelWorkerMessageValidators
+from mms.utils.model_server_error_codes import ModelServerErrorCodes
+
+def test_validate_load_message_missing_model_path():
+    invalid_msg = '{ \"modelName\" : \"some-model-name\" }'
+    with pytest.raises(MMSError) as error:
+        ModelWorkerMessageValidators.validate_load_message(invalid_msg)
+
+    assert error.value.get_code() == ModelServerErrorCodes.INVALID_LOAD_MESSAGE, 'error codes don\'t match'
+    assert error.value.get_message() == 'Load command missing \"model-path\" key'
+
+    valid_msg = '{ \"modelName\" : \"some-model-name\", \"modelPath\" : \"some-model-path\" }'
```

Comment:
Would be better if each testcase had only one scenario. For example, one test case for negative scenario and another for positive scenario.

Reply:
Okay. Will make the required change.

Reply:
Done in 3b280a1a8186c3a2e9a49c9d87f9320a26805ec2